src/game.cpp

```diff
@@ -3677,6 +3677,9 @@ void Game::handlePointingAtNode(GameRunData *runData,
 				if (playeritem_def.node_placement_prediction == "" ||
 						nodedef_manager->get(map.getNodeNoEx(nodepos)).rightclickable)
 					client->interact(3, pointed); // Report to server
+				else
+					soundmaker->m_player_rightpunch_sound =
+						playeritem_def.sound_place_failed;
```

Comment:
Oh jeez, please add brackets around the bodies of this if..else statement, please, I feel it would help clarify things a lot.